lib/screens/shuttle_schedule.dart

```diff
@@ -17,375 +20,160 @@ class _ShuttleScheduleState extends State<ShuttleSchedule> {
   bool isSelectedSGWToLoy = false;
   bool isSelectedLoyToSGW = false;
 
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(
-          ConcordiaLocalizations.of(context).shuttle,
+  dynamic getSchedule() async {
+    return json
+        .decode(await rootBundle.loadString('assets/shuttleSchedule.json'));
+  }
```

Comment:
Probably should use the ShuttleData provider we already have by moving any logic or parsing from here to ShuttleData and invoke it's methods using the provider.

Reply:
Addressed in 46970d74438e8a9a7892b32b8e26c00cf6f03f58